src/main/java/spoon/reflect/visitor/chain/CtQueryImpl.java

```diff
@@ -30,6 +30,11 @@
  */
 public class CtQueryImpl implements CtQuery {
 
+	/**
+	 *
+	 */
```

Comment:
Is this done to pass Qodana or checkstyle checks? I am unsure why it is required. Type members with `private` modifiers are not part of the public API and can be changed at will. Thus, documenting them should not be necessary.

Reply:
no, it was a mistake by my IDE, that expected a comment there. But I really don't know any code comment for this.